lib/globber.js

```diff
@@ -1,29 +1,33 @@
 'use strict';
 const glob = require('glob');
 const Rx = require('rx');
+const fs = require('fs');
 
 function globber(paths) {
   const files = new Rx.Subject();
   files.fileEvents = [];
 
-  const Glob = glob.Glob;
   let doneCount = 0;
 
-  paths.forEach(function (path) {
-    const fileEvents = new Glob(path);
+  function globFiles(path) {
+    const fileEvents = new glob.Glob(path);
 
     fileEvents.on('match', function (file) {
-      files.onNext(file);
+      if (fs.statSync(file).isFile()) {
```

Comment:
Glob has an option, "nodir", which will prevent it from matching directories. I think adding that option would do the same thing as this PR?